tsconfig.json

```diff
@@ -16,6 +16,7 @@
         /* Specify how TypeScript looks up a file from a given module specifier. */
         "outDir": "./dist",
         /* Specify an output folder for all emitted files. */
+        "resolveJsonModule": true,
```

Comment:
nit: do we actually need this here? I see `tsconfig.json` in `server` package has already this config.